main.go

```diff
@@ -39,41 +61,58 @@ func main() {
 		os.Exit(1)
 	}
 
-	for _, singlePackage := range packages {
-		vulnerabilitiesOSS, err := ossFetcher.Test(singlePackage.Name, singlePackage.Version)
-		if err != nil {
-			fmt.Println(err.Error())
-			os.Exit(1)
-		}
+	vulnerabilitiesOSS, err := ossFetcher.TestAll(packages)
+	if err != nil {
+		fmt.Println(err.Error())
```

Comment:
This should go to the logger instead of Stdout.

Reply:
Not sure if I am 100% right. Just checking.

Reply:
you are right, this has already been updated in #15, I can fix it of course if you think it is important to have a clean master, or I can leave as is, as this is minor, awaiting for #15 to be merged.
What do you think ?